### PR TITLE
Replaced aws analytics package

### PR DESCRIPTION
**Pull request by omenocal**
The `aws-sdk-mobile-analytics` package seemed to have several issues when trying to stop tracking, so I'm replacing it for the very own AWS package.

### CODE REVIEW

**Review by wuelcas — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -45,20 +45,20 @@ skill.onState('my-state', (alexaEvent) => {
 })
 ```
 
-### Logging variables
+### Logging custom variables
 You can also add additional values which will be logged along with the state custom event
 ```javascript
 skill.onState('my-state', (alexaEvent) => {
-  alexaEvent.ama.variables.myVariable = 'hello';
+  alexaEvent.ama.logEvents('response', 'hello');
```
Comment (armonge) [inline README.md]: why the change in the interface?

File: README.md [hunk 2]
```diff
@@ -45,20 +45,20 @@ skill.onState('my-state', (alexaEvent) => {
...
 })
 ```
 
-### Custom Events
+### Specific Events
 You can also log custom events from a state.
 ```javascript
 skill.onState('my-state', (alexaEvent) => {
-  alexaEvent.ama.log('my-custom-event', { myVariable: 'hello' });
+  alexaEvent.ama.logEvents('my-custom-event', { myVariable: 'hello' });
```
Comment (armonge) [inline README.md]: what's the difference between this and the `custom variables`?

File: tests/event-rider.spec.js [hunk 3]
```diff
@@ -1,73 +0,0 @@
-'use strict';
```
Comment (armonge) [inline tests/event-rider.spec.js:1]: seems like we got rid of a lot of tests, but coverage dropped

File: package.json [hunk 4]
```diff
@@ -45,7 +45,7 @@
     "mocha": "^3.2.0",
     "mocha-jenkins-reporter": "^0.3.7",
     "simple-mock": "^0.7.3",
-    "voxa": "^2.2.0"
+    "voxa": "^2.2.1"
```
Comment (armonge) [inline package.json:48]: i was just thinking we should start looking into peer dependencies, thoughts?

@rmberrios @wuelcas @wuelcas 
https://yarnpkg.com/lang/en/docs/dependency-types/#toc-peerdependencies

**Review by armonge — CHANGES_REQUESTED** [review]
In general i don't feel comfortable changing the interface of a published package. Is there a reason for this? 

I know we changed internally to use aws-sdk instead of aws-sdk-mobile-analytics . Does that *really* means we need to change the package interface to the developer

File: tests/event-rider.spec.js (hunk 3 above)
Comment (omenocal) [reply tests/event-rider.spec.js:1]: Yes, I'll add more

File: README.md (hunk 2 above)
Comment (omenocal) [reply README.md]: This is an event with its own variables. Like

```
RecipeIntent
recipe: chicken
```

File: README.md (hunk 1 above)
Comment (omenocal) [reply README.md]: This would be part of a Custom event:
```
Custom
response: hello
```

Comment (omenocal) [reply README.md]: I just realized the documentation mentions a lot the word `events` and it'd be great to `logEvents` and not just `log`. Since the name `log` sounds like it will write a log somewhere, like the debug or console.log methods.

File: package.json (hunk 4 above)
Comment (omenocal) [reply package.json:48]: I don't think `peerDependencies` would work in this specific project. I am using voxa 2.2.1 for this package because is public. For the simon skill (the project we're testing this package into, I'm using a private version of voxa. Maybe in the future ?

**Review by armonge — APPROVED** [review]
